purchase_report_custom_hls/models/purchase_order.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+
+from odoo import fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    estimated_time_arrival = fields.Char("Estimated Time of Arrival(ETA)")
+    estimated_time_departure = fields.Char("Estimated Time of Departure(ETD)")
```

Comment:
I'd suggest changing the fields to `eta_date` and `etd_date`.  ETD/ETA are generic enough.

Reply:
Thank you.
I renamed fields.